torchvision/transforms/functional_tensor.py

```diff
@@ -1046,71 +1023,60 @@ def _get_gaussian_kernel1d(kernel_size: int, sigma: float):
     return kernel1d
 
 
-def _get_gaussian_kernel2d(kernel_size: List[int], sigma: List[float]):
-    ksize_x, ksize_y = kernel_size
-    sigma_x, sigma_y = sigma
-
-    kernel1d_x = _get_gaussian_kernel1d(ksize_x, sigma_x)
-    kernel1d_y = _get_gaussian_kernel1d(ksize_y, sigma_y)
+def _get_gaussian_kernel2d(
+        kernel_size: List[int], sigma: List[float], dtype: torch.dtype, device: torch.device
+) -> Tensor:
+    kernel1d_x = _get_gaussian_kernel1d(kernel_size[1], sigma[0]).to(device, dtype=dtype)
+    kernel1d_y = _get_gaussian_kernel1d(kernel_size[0], sigma[1]).to(device, dtype=dtype)
```

Comment:
Should it be `kernel_size[0]` for `kernel1d_x` and `kernel_size[1]` for `kernel1d_y`?

Reply:
Yes, you are right, I was thinking to switch the convention but then decided to keep it as `(kx, ky)`. Can you fix that please ?